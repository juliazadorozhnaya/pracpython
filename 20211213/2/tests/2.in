[322, 77, 343, 406, 91, 679, 245, 959, 266, 882, 448, 448, 518, 511, 952, 294, 560, 329, 98, 763, 875, 259, 7, 700, 441, 98, 21, 203, 504, 112, 931, 462, 595, 238, 567, 259, 574, 693, 21, 399, 70, 63, 217, 504, 728, 777, 182, 392, 42, 196, 70, 392, 819, 805, 567, 707, 973, 728, 476, 742, 966, 777, 322, 259, 595]
