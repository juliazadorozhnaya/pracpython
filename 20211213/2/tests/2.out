[7, 21, 21, 42, 63, 70, 70, 77, 91, 98, 98, 112, 182, 196, 203, 217, 238, 245, 259, 259, 259, 266, 294, 322, 322, 329, 343, 392, 392, 399, 406, 441, 448, 448, 462, 476, 504, 504, 511, 518, 560, 567, 567, 574, 595, 595, 679, 693, 700, 707, 728, 728, 742, 763, 777, 777, 805, 819, 875, 882, 931, 952, 959, 966, 973]
